inception/encoder.go

```diff
@@ -266,6 +267,38 @@ func getGetInnerValue(ic *Inception, name string, typ reflect.Type, ptr bool, fo
 		out += "buf.Write(obj)" + "\n"
 	case reflect.Map:
 		out += getMapValue(ic, ptname, typ, ptr, forceString)
+	case reflect.Struct:
+		if typ.Name() == "" {
+			ic.q.Write("{")
+			ic.q.Write(" ")
```

Comment:
This is to avoid issues if there is no content, or everything is omitted by omitempty. This allows us to be able to avoid runtime checks to see if anything was written.

If something was written, the last comma will be removed (and the space will be at the start). If nothing was written, it will remove the space. 

The same thing happens for ordinary structs. See [line 497](https://github.com/klauspost/ffjson/blob/inline-structs-resubmit/inception/encoder.go#L497) in the same file.